l10n_ar_txt_sire/models/account_journal.py

```diff
@@ -0,0 +1,324 @@
+from odoo import models, fields, _
+from odoo.exceptions import RedirectWarning, UserError
+import re
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    settlement_tax = fields.Selection(selection_add=[
+        ('sire', 'TXT Retenciones SIRE'),
+        ('certificado_retencion_impositiva', 'TXT Certificado Retención Impositiva'),
+    ])
+
+    def sire_files_values(self, move_lines):
+        """ Devuelve contenido del archivo Retenciones_sire.txt . Implementado según especificación de tarea 40906.
+        https://www.afip.gob.ar/sire/documentos/SIRE-especificacion-para-emision-por-lote.pdf apartado 3.
+        F2003 CERTIFICADOS SUJETOS DOMICILIADOS EN EL EXTERIOR.
+        También se puede ver la especificación en doc/SIRE-especificacion-para-emision-por-lote.pdf """
+        self.ensure_one()
+        content = ''
+        # VALIDACIONES
+        self._sire_validations(move_lines)
+        for line in move_lines.sorted(key=lambda r: (r.date, r.id)):
+            payment = line.payment_id
+            pais = line.partner_id.country_id
+            es_persona = line.partner_id.company_type == 'person'
+            company_vat = payment.company_id.vat
+
+            fecha_impuesto = fields.Date.from_string(line.date).strftime('%d/%m/%Y')
+            # 1 Formulario (integer long 4, 1-4, obligatorio) --> '2003' comentado por agi en tarea 40050, en nota del
+            # 23/10/2024 a las 10:00hs
+            # Además en la sección "3.2. F2003 - Validaciones", pàgina 5 del pdf de la especificación se indica que
+            # debe ser fijo "2003"
+            content += '2003'
+            # 2 Versión (integer long 4, 5-8, obligatorio) --> '0100' comentado por agi en tarea 40050, en nota del
```

Comment:
make the comments smaller possible
```suggestion
            # 2 Versión (integer long 4, 5-8, obligatorio) --> '0100' task 40050 agi comment 2024.10.23 10:00
```